docs/coordinates/performance.inc.rst

```diff
@@ -56,3 +56,57 @@ properties::
 
 ..
   EXAMPLE END
+
+Improving Performance for Arrays of ``obstime``
+===============================================
+
+The most expensive operations when transforming between observer-dependent coordinate
+frames (e.g. ``AltAz``) and sky-fixed frames (e.g. ``ICRS``) are the calculation
+of astrometric values like precession and nutation matrices.
+
+If |skycoord| instances with a large ``obstime`` array are transformed,
+these calculations can be speed up by factors up to 100 and still keeping micro-arcsecond resolution
```

Comment:
I would keep the up too, since for more conservative time resolutions it is a factor of 10, it really depends on the density of the obstimes and the chosen time resolution.

Reply:
I took the other part of your suggestion, though